packages/roosterjs-content-model/lib/formatHandlers/list/listItemElementFormatHandler.ts

```diff
@@ -0,0 +1,19 @@
+import { ContentModelListItemFormat } from '../../publicTypes/format/ContentModelListItemFormat';
+import { FormatHandler } from '../FormatHandler';
+
+/**
+ * @internal
+ */
+export const listItemElementFormatHandler: FormatHandler<ContentModelListItemFormat> = {
+    parse: (format, element) => {
+        const alignment = element.style.alignSelf;
+        if (alignment && ['end', 'start', 'center'].indexOf(alignment) > -1) {
```

Comment:
I think all the logic here is already included in directionFormatHandler, right? Then we don't need this handler at all.